content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+---
+title: Create a Cold-Tiered Volume and use HPE Ezmeral Data Fabric Object Store
```

Comment:
Hi Dale, thank you for your review and suggestions.
The words "cold-tiered", and "volume" are indeed not proper nouns. I use the title case because they are concepts in HPE Ezmeral Data Fabric.
However, these concepts should also exist in products other than HPE and may have different names. But my purpose is really mainly for emphasis.
Next, I will only use title case for proper nouns, I think audiences should be aware of the unique concepts in the product.